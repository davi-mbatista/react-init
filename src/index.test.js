test('It renders without a App', () => {
    expect('hello!').toMatchSnapshot();
});
