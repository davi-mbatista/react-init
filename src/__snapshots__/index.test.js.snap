// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders without a App 1`] = `"hello!"`;
